.circleci/config.yml

```diff
@@ -1,242 +1,222 @@
-
-run_karma_tests: &run_karma_tests
-  run:
-    name: Run karma tests
-    command: yarn sauce --coverage
-    working_directory: packages/integration-karma
-
-run_karma_tests_native: &run_karma_tests_native
-  run:
-    name: Run karma tests - native shadow
-    command: yarn sauce --native-shadow --coverage
-    working_directory: packages/integration-karma
-
-run_karma_tests_compat: &run_karma_tests_compat
-  run:
-    name: Run karma tests - compat
-    command: yarn sauce --compat --coverage
-    working_directory: packages/integration-karma
-
-run_karma_coverage: &run_karma_coverage
-  run:
-    name: Compute karma coverage
-    command: yarn coverage
-    working_directory: packages/integration-karma
-
-run_integration_tests_chrome: &run_integration_tests_chrome
-  run:
-    name: Run integration test - Headless Chrome
-    command: yarn sauce:prod --browsers chrome
-    working_directory: packages/integration-tests
-
-run_integration_tests_ie11: &run_integration_tests_ie11
-  run:
-    name: Run integration test - IE11 SauceLabs
-    command: yarn sauce:prod_compat --browsers ie11
-    working_directory: packages/integration-tests
-
-# BEST
-setup_best_environment: &setup_best_environment
-  run:
-    name: Setup BEST environment
-    command: |
-        echo 'export PULL_REQUEST=${CIRCLE_PULL_REQUEST}' >> $BASH_ENV
-        echo 'export REPO_NAME=${CIRCLE_PROJECT_REPONAME}' >> $BASH_ENV
-        echo 'export TARGET_COMMIT=${CIRCLE_SHA1}' >> $BASH_ENV
-        echo 'export BASE_COMMIT=`git rev-parse origin/master`' >> $BASH_ENV
-
-        echo -e "$GIT_APP_CERT" | base64 -d >> ~/lwc/git_app.pem
-
-run_best_chrome: &run_best_chrome
-  run:
-    name: Run BEST
-    working_directory: packages/perf-benchmarks
-    command: yarn start --projects best.headless.config.js --externalStorage=@best/store-aws --runner remote --runInBatch --dbAdapter=sql/postgres --dbURI=${BEST_HUB_DATABASE_URL}
-    no_output_timeout: 40m
-
-compare_best_results: &compare_best_results
-  run:
-    name: Comparing Benchmarks
-    working_directory: packages/perf-benchmarks
-    command: yarn start --projects best.headless.config.js --compareStats ${BASE_COMMIT} ${TARGET_COMMIT} --externalStorage=@best/store-aws --gitIntegration
-
-store_best_artifacts: &store_best_artifacts
-  store_artifacts:
-    path: ~/lwc/packages/perf-benchmarks/__benchmark_results__/
-    destination: benchmarks
+# Shared config
+ignore_forks: &ignore_forks
+  branches:
+    # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
+    ignore: /pull\/[0-9]+/
 
 
 # Jobs definition
-version: 2
-jobs:
-  build:
-    <<: *env_default
-    steps:
-      - checkout
-      - *restore_yarn_cache
-      - *install_and_build
-      - *save_yarn_cache
+version: 2.1
 
-      - *run_sanity_checks
-      - *run_linting
-      - *run_format
 
-      - *save_workspace
+# Custom executors definition
+executors:
+  node:
+    working_directory: ~/lwc
+    docker:
+      - image: circleci/node:12
```

Comment:
```suggestion
  node_maintenance_lts:
    working_directory: ~/lwc
    docker:
      - image: circleci/node:10
  node_active_lts:
    working_directory: ~/lwc
    docker:
      - image: circleci/node:12
```

Reply:
I was planning to do this in a separate PR. Will merge this and open a new branch for this.